### PR TITLE
docs: add details on ssh key + pull existing prereq docs

### PR DESCRIPTION
**Pull request by vburckhardt**
### Description

- Additional details on ssh key (as question keeps coming)
- Pull remaining existing doc on account setup

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [ ] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

**Review by argeiger — DISMISSED** [review]

**Comment by SirSpidey** [comment]
@vburckhardt Created #176 for the readme.

**Review by SirSpidey — DISMISSED** [review]
👍 for content. This stuff will likely move to Cloud Docs when we have a place for solution guides.

File: README.md [hunk 1]
```diff
@@ -38,9 +38,51 @@ For more information about the default configuration, see [Default Secure Landin
...
+### Configure your IBM Cloud account for Secure Landing Zone
+
+1.  Log in to [IBM Cloud](https://cloud.ibm.com) with the IBMid you used to set up the account. This IBMid user is the account owner and has full IAM access.
+1.  [Complete the company profile](https://cloud.ibm.com/docs/account?topic=account-contact-info) and contact information for the account. This profile is required to stay in compliance with IBM Cloud Financial Service profile.
+1.  [Enable the Financial Services Validated option](https://cloud.ibm.com/docs/account?topic=account-enabling-fs-validated) for your account.
+1.  Enable virtual routing and forwarding (VRF) and service endpoints by creating a support case. Follow the instructions in  [enabling VRF and service endpoints](https://cloud.ibm.com/docs/account?topic=account-vrf-service-endpoint&interface=ui#vrf).
+
+### Set up account access (Cloud IAM)
+
+1.  If you are provisioning resources manually, create an IBM Cloud [API key](https://cloud.ibm.com/docs/account?topic=account-userapikey#create_user_key). The user who owns this key must have the Administrator role.
```
Comment (SirSpidey) [inline README.md:60]: @vburckhardt: Can we be more specific about what `provisioning resources manually` means? Is it outside of the catalog through Terraform directly or something else?

File: README.md [hunk 2]
```diff
@@ -38,9 +38,51 @@ For more information about the default configuration, see [Default Secure Landin
...
+
+1.  Create the service instance:
+
+    1.  (Optional) [Create a resource group](https://cloud.ibm.com/docs/account?topic=account-rgs&interface=ui) for your instance.
+    1.  On the Hyper Protect Crypto Services (https://cloud.ibm.com/catalog/services/hyper-protect-crypto-services) details page, select a plan.
+    1.  Complete the required details that are required and click **Create**.
+
+1.  Initialize Hyper Protect Crypto Services:
+
+    To initialize the provisioned Hyper Protect Crypto Service instance, follow the steps in [Getting started with IBM Cloud Hyper Protect Crypto Services](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-get-started) and perform the quick initialization.
```
Comment (SirSpidey) [inline README.md:79]: @vburckhardt I don't see any mention of a quick initialization. I see 3 options in the docs about initialization. We should point to one of those, or use the term that they can search for on the docs page.

Comment (argeiger) [reply README.md:79]: Yea looks like they changed to the docs.  I would just take the "and perform the quick initialization" out

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@vburckhardt The common-dev-assets git submodule version in this Pr is very old. Run `git submodule update --rebase` to update it to the version currently on `main` branch. (We have a new pre-commit hook to catch this going forward)

File: README.md (hunk 1 above)
Comment (vburckhardt) [reply README.md]: I removed the sentence, it did not fully make sense in first place

**Review by SirSpidey — APPROVED** [review]
👍 Content LGTM

**Comment by ocofaigh** [comment]
`terraform-config-inspect` hook is failing. You may have an old version installed locally Vincent. Try running `make` locally to ensure you have same version the pipeline is running

**Review by ocofaigh — APPROVED** [review]

**Review by SirSpidey — APPROVED** [review]
👍

**Review by argeiger — DISMISSED** [review]

**Pull request merged**